### PR TITLE
Update device trust docs with Linux support

### PR DESCRIPTION
**Pull request by codingllama**
Add Linux to the supported OSes, mention it where appropriate and add a troubleshooting item for lack of TPM permissions.

https://github.com/gravitational/teleport.e/issues/827

### CODE REVIEW

File: docs/pages/access-controls/device-trust/device-management.mdx [hunk 1]
```diff
@@ -58,56 +55,31 @@ $ tsh device asset-tag
...
-      ...
-    }
+    $ tsh device asset-tag
+    (=devicetrust.asset_tag=)
     ```
   </TabItem>
 </Tabs>
 </Details>
-Use the serial number to register device:
-<Tabs>
```
Comment (codingllama) [inline docs/pages/access-controls/device-trust/device-management.mdx:72]: I've dropped the tabs here and instead made the `--os` value a variable, to call attention to what is actually the only distinction between the blocks.

**Comment by codingllama** [comment]
Rendering looks alright to me.

File: docs/pages/includes/device-trust/enroll-troubleshooting.mdx [hunk 2]
```diff
@@ -12,3 +12,14 @@ Teleport as such. Follow the [registration](
 [enrollment](
 ../../access-controls/device-trust/device-management.mdx#enroll-a-trusted-device) steps
 and make sure to `tsh logout` and `tsh login` after enrollment is done.
+
+### "Failed to open the TPM device" on Linux
+
+Linux users need permissions to read and write from the TPM device, /dev/tpmrm0.
```
Comment (ptgott) [inline docs/pages/includes/device-trust/enroll-troubleshooting.mdx:18]:
```suggestion
Linux users need permissions to read and write from the TPM device, `/dev/tpmrm0`.
```

**Review by ptgott — APPROVED** [review]
Approved with a minor nit

**Comment by ptgott** [comment]
@codingllama Is this change coming in v15, or should we backport this PR?

**Comment by codingllama** [comment]
It's only for v15, no need to backport

**Review by rosstimothy — APPROVED** [review]

**Pull request merged**